src/tests/ftest/ior/ior_intercept_dfuse_mix.py

```diff
@@ -42,78 +43,112 @@ def test_ior_intercept_dfuse_mix(self):
-        self.run_multiple_ior_with_pool(with_intercept, intercept)
-        self.log_metrics(without_intercept, with_intercept)
+        client_count = len(self.hostlist_clients)
+        w_clients = self.hostlist_clients[0:client_count / 2]
+        wo_clients = self.hostlist_clients[client_count / 2:]
+        self.run_ior_threads_il(
+            results=results, intercept=intercept, with_clients=w_clients,
+            without_clients=wo_clients)
+
+        # Print the raw results from the IOR stdout.
+        IorCommand.log_metrics(
+            self.log, "{} clients - with interception library".format(
+                len(w_clients)), results[1])
+        IorCommand.log_metrics(
+            self.log, "{} clients - without interception library".format(
+                len(wo_clients)), results[2])
+
+        # Get Max, Min, and Mean throughput values for Write and Read.
+        w_write_results = results[1][0]
+        w_read_results = results[1][1]
+        wo_write_results = results[2][0]
+        wo_read_results = results[2][1]
 
         max_mib = int(IorMetrics.Max_MiB)
         min_mib = int(IorMetrics.Min_MiB)
         mean_mib = int(IorMetrics.Mean_MiB)
 
+        w_write_max = float(w_write_results[max_mib])
+        wo_write_max = float(wo_write_results[max_mib])
+        w_write_min = float(w_write_results[min_mib])
+        wo_write_min = float(wo_write_results[min_mib])
+        w_write_mean = float(w_write_results[mean_mib])
+        wo_write_mean = float(wo_write_results[mean_mib])
+
+        w_read_max = float(w_read_results[max_mib])
+        wo_read_max = float(wo_read_results[max_mib])
+        w_read_min = float(w_read_results[min_mib])
+        wo_read_min = float(wo_read_results[min_mib])
+        w_read_mean = float(w_read_results[mean_mib])
+        wo_read_mean = float(wo_read_results[mean_mib])
+
+        # Calculate the increase for the 6 values.
+        write_max_change = -1
+        if wo_write_max > 0:
+            write_max_change = round(w_write_max / wo_write_max, 4)
+        write_min_change = -1
+        if wo_write_min > 0:
+            write_min_change = round(w_write_min / wo_write_min, 4)
+        write_mean_change = -1
+        if wo_write_mean > 0:
+            write_mean_change = round(w_write_mean / wo_write_mean, 4)
+
+        read_max_change = -1
+        if wo_read_max > 0:
+            read_max_change = round(w_read_max / wo_read_max, 4)
+        read_min_change = -1
+        if wo_read_min > 0:
+            read_min_change = round(w_read_min / wo_read_min, 4)
+        read_mean_change = -1
+        if wo_read_mean > 0:
+            read_mean_change = round(w_read_mean / wo_read_mean, 4)
+
+        # Print the summary of improvements.
+        self.log.info(
+            "--- Throughput Improvement with Interception Library ---")
+        self.log.info("Clients with IL: {}".format(w_clients))
+        self.log.info("Clients without IL: {}\n".format(wo_clients))
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions